include/constants/rgb.h

```diff
@@ -1,6 +1,10 @@
 #ifndef GUARD_RGB_H
 #define GUARD_RGB_H
 
+#define R(color) ((color) & 0x1F)
+#define G(color) (((color) >> 5) & 0x1F)
+#define B(color) (((color) >> 10) & 0x1F)
```

Comment:
These names imply to-RGB, but they're from-RGB. i.e. I'd expect `R(x) | G(y) | B(z)` to be equivalent to `RGB(x, y, z)`